manifests/http_port.pp

```diff
@@ -1,24 +1,48 @@
 define squid::http_port (
-  Variant[Pattern[/\d+/], Integer]
-          $port    = $title,
-  Boolean $ssl     = false,
-  String  $options = '',
-  String  $order   = '05',
+  Optional[Stdlib::Port] $port    = undef,
+  Optional[String[1]]    $host    = undef,
```

Comment:
There is `Stdlib::Fqdn`

Reply:
I made it `Stdlib::Host`, so that IP addresses are allowed.